core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/heuristic/selector/move/generic/list/KOptCycleInfo.java

```diff
@@ -0,0 +1,23 @@
+package org.optaplanner.core.impl.heuristic.selector.move.generic.list;
```

Comment:
Since there is plenty of classes related to this particular move, I'd consider creating a `...list.kopt` subpackage for all of them.